formspree/forms/helpers.py

```diff
@@ -8,7 +8,7 @@
 
 from formspree import settings
 
-HASH = lambda x, y: hashlib.md5(x+y+settings.NONCE_SECRET).hexdigest()
+HASH = lambda x, y: hashlib.md5(x.encode('utf-8')+y.encode('utf-8')+settings.NONCE_SECRET).hexdigest()
```

Comment:
That looks to be fine to me, but I haven't done much in terms of UTF-8 encoding, so I can't say for certain.